frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -19,8 +49,142 @@ function AuthedView() {
     <dl>
       <h1>
         <MetaTitle title="Dashboard" />
-        Welcome, <span style={{ color: "dodgerblue" }}>{user.username}</span>
+        <ProjectList />
       </h1>
     </dl>
   );
 }
+
+const projects = [
+  {
+    id: 1,
+    name: "ZaneOps",
+    updated_at: "Jan 13, 2024",
+    status: "0/5 Services Up",
+    actions: "Settings",
+    tracker: 0
+  },
+  {
+    id: 2,
+    name: "ZaneOps",
+    updated_at: "Jan 13, 2024",
+    status: "5/5 Services Up",
+    actions: "Settings",
+    tracker: 1
+  },
+  {
+    id: 3,
+    name: "ZaneOps",
+    updated_at: "Jan 13, 2024",
+    status: "2/5 Services Up",
+    actions: "Settings",
+    tracker: 2
+  }
+];
+
+type TrackerColor = "red" | "green" | "yellow";
+function getBadgeColor(tracker: number): TrackerColor {
+  switch (tracker) {
+    case 0:
+      return "red";
+    case 1:
+      return "green";
+    case 2:
+      return "yellow";
+    default:
+      return "green";
+  }
+}
```

Comment:
This is temporary, you might forgot it when you add the logic... so maybe add a `// TODO: to remove` comment here ? 🤔